operations/rates.md

```diff
@@ -287,183 +227,210 @@ The price in the response is dependent on the enterprise's [pricing](configurati
+| `EmptyUnitAdjustment` | number | required | Price adjustment for when the resource booked with this rate is not full to capacity. |
+| `ExtraUnitAdjustment` | number | required | Price adjustment for when the resource booked with this rate exceeds capacity. |
+| ~~`DatesUtc`~~ | ~~array of string~~ | ~~optional~~ | **Deprecated!** Use `TimeUnitStartsUtc` instead.|
+| ~~`BasePrices`~~ | ~~array of number~~ | ~~required~~ | **Deprecated!** Use `BaseAmountPrices` instead.|
+
+#### Category pricing
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `CategoryId` | string | required | Unique identifier of the category. |
+| `AmountPrices` | array of [Amount](_objects.md#amount) | required | Prices of the rate for the resource category in the covered dates. |
+| ~~`Prices`~~ | ~~array of number~~ | ~~required~~ | ~~Prices of the rate for the resource category in the covered dates.~~ **Deprecated!** Use `AmountPrices` instead.|
 
-#### Resource category pricing
+#### Category adjustment
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `CategoryId` | string | required | Unique identifier of the [Resource category](resources.md#resource-category). |
-| `AmountPrices` | array of [Amount value](accountingitems.md#amount-value) | required | Prices of the rate for the resource category in the covered dates. |
+| `CategoryId` | string | required | Unique identifier of the adjustment category. |
+| `ParentCategoryId` | string | optional | Unique identifier of the parent category that serves as a base price for the current category. |
+| `AbsoluteValue` | number | required | Absolute value of the adjustment (e.g. `50` represents 50 EUR in case the rate currency is `EUR`). |
+| `RelativeValue` | number | required | Relative value of the adjustment (e.g. `0.5` represents 50% increase). |
 
-#### Resource category adjustment
+#### Age category adjustment
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `CategoryId` | string | required | Unique identifier of the adjustment [Resource category](resources.md#resource-category). |
-| `ParentCategoryId` | string | optional | Unique identifier of the parent [Resource category](resources.md#resource-category) that serves as a base price for the current category. |
-| `RelativeValue` | number | required | Relative value of the adjustment \(e.g. `0.5` represents 50% increase\). |
-| `AbsoluteValue` | number | required | Absolute value of the adjustment \(e.g. `50` represents 50 EUR in case the rate currency is `EUR`\). |
+| `AgeCategoryId` | string | required | Unique identifier of the age category. |
+| `AbsoluteValue` | number | required | Absolute value of the adjustment (e.g. `50` represents 50 EUR in case the rate currency is `EUR`). |
+| `Type` | [Age category adjustment type](rates.md#age-category-adjustment-type) | required |  |
```

Comment:
Missing description

Reply:
fixed as a part of this [commit](https://github.com/MewsSystems/gitbook-connector-api/pull/635/commits/aea81f3fe83c61522fc3bcfd401a5535459b7ab0)